src/io_tools/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
 add_library(io_tools read_nrg.cpp write_nrg.cpp) 
 target_include_directories(io_tools PRIVATE ${CMAKE_SOURCE_DIR}) 
 target_include_directories(io_tools PRIVATE ${CMAKE_SOURCE_DIR}/../external/kdtree/) 
+target_include_directories(io_tools PRIVATE ${CMAKE_SOURCE_DIR}/tools/)
```

Comment:
As a note I would not enforce a flat header space, it is difficult to sustain as your project grows.